testing/trino-testing-services/src/main/java/io/trino/testng/services/ManageTestResources.java

```diff
@@ -88,7 +89,10 @@ private static boolean isEnabled()
         if (System.getProperty("ManageTestResources.enabled") != null) {
             return Boolean.getBoolean("ManageTestResources.enabled");
         }
-        if (System.getenv("DISABLE_REPORT_RESOURCE_HUNGRY_TESTS_CHECK") != null) {
+        if (getenv("DISABLE_REPORT_RESOURCE_HUNGRY_TESTS_CHECK") != null) {
+            return false;
+        }
+        if (getenv("TESTCONTAINERS_REUSE_ENABLE") != null) {
             return false;
         }
```

Comment:
The listener reports a failure when the tests end and doesn't actually prevent a test from running or being green.

Anyway this is a preexisting thing so it needs to be a separate commit with some explanation/justification of the change in the commit message.

Reply:
Also FYI @findepi as he's the original author of the check so might have opinions.